src/google/google-calendars/googlecalendarsyncadaptor.cpp

```diff
@@ -99,11 +101,45 @@ void setGhostEventCleanupPerformed()
 
 QString gCalEventId(KCalCore::Incidence::Ptr event)
 {
-    return event->customProperty("jolla-sociald", "gcal-id");
+    // we abuse the comments field to store our gcal-id.
+    // we should use a custom property, but those are deleted on incidence deletion.
```

Comment:
The alternative to abusing the comments field is to do: https://github.com/mer-packages/mkcal/pull/22

But, this way I think is probably fine as-is (since it means less messing with mkcal which could break unit tests or have other behavioural knock-on effects such as other queries now returning more results than expected, etc).